lib/cluster/election.c

```diff
@@ -349,26 +359,28 @@ election_check(election_t *e)
             const crm_node_t *node;
             char *key = NULL;
 
+            crm_warn("Received too many votes in %s", e->name);
             g_hash_table_iter_init(&gIter, crm_peer_cache);
             while (g_hash_table_iter_next(&gIter, NULL, (gpointer *) & node)) {
                 if (crm_is_peer_active(node)) {
-                    crm_err("member: %s proc=%.32x", node->uname, node->processes);
+                    crm_warn("* expected vote: %s", node->uname);
```

Comment:
At this point, just noting how pleasant it was for me recently to
quickly discern messages about resource state changes thanks to
` * ` sequence being easily identifiable in the surrounding noise,
hence I am afraid using it excessively would harm that subjective
benefit.

Reply:
I've been staring at these logs a lot lately :) and this format really helps. It makes it clear that multiple log lines relate to each other. Also this particular situation is very rare.

+ and - are used in CIB diffs, so I'd rather not reuse those for anything else, since those diffs are so common in the logs.